Readme.md

```diff
@@ -6,23 +6,23 @@
 <!--
 [![Test Coverage][coveralls-image]][coveralls-url]
  -->
-Utility to parse a string bytes (ex: `1TB`) to bytes (`1,000,000,000,000`) and vice-versa.
+Utility to parse a size string in bytes (e.g. `'1kB'`, `'2KiB'`) to numeric (`1000`, `2048`) and vice-versa.
 
-This uses the byte units defined in [ISO/IEC 80000-13:2008][iec], both the binary prefixes and the original SI units.
+This is a fork of the [bytes][bytes] module, except it
 
-This is a fork of the [bytes][bytes] module, except:
+ * supports all of both the binary and decimal prefixes defined by [ISO/IEC 80000-13:2008][iec]
+ * supports [JEDEC][jedec], a compatibility mode resembling the upstream original
+ * uses decimal units by default, which can be overridden per call or by default
 
- * It uses IEC units by default
- * Supports a wider range of units
- * Supports changing to compatability ([JEDEC][jedec]) mode, and formatting in whichever prefix type you prefix (binary, metric, jedec)
-
- TypeScript definitions included.
+TypeScript definitions included.
 
 ## Supported Units
 
-Supported units and abbreviations are as follows and are case-insensitive:
+Supported units are as follows and are case-insensitive. Note that only the abbreviation will be parsed/formatted, the full names are for the reader's understanding only.
+
+### Decimal
 
-### Metric/Decimal Prefixes
+Also referred to as metric, SI, IEC, and ISO.
```

Comment:
I'm not sure how correct this formally is, especially calling it ISO since I'd expect the binary units to be ISO-defined as well, but the whole subject of colloquial names is fluid anyway so meh

Reply:
This is better to call it Metric by default.

It's not really called "IEC" since that defines the Binary units, and re-prioritizes the original SI-metric units over JEDEC.  Nor would it be called ISO for a similar reason.

Metric, Decimal, SI, all work

Reply:
Thanls for clarifying. So Metric as heading, and decimal and SI as synonyms?

Reply:
yup, though it might be nice to relink the IEC article from above (and the JEDEC in the compatibility one) plus maybe find+link the binary one as well